getting_started/access_your_cluster.adoc

```diff
@@ -0,0 +1,112 @@
+through the following paths:
+
+* Cluster ID: The unique cluster name provided by the customer during provisioning. It is lowercase, and only contains letters, numbers, and hyphens.
+
+* Console URL: The {product-title} URL for the
+xref:../architecture/infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web
+console].
++
+----
+https://console.<cluster-id>.openshift.com
+----
+
+* API URL: The {product-title} URL for the OpenShift and Kubernetes
+xref:../rest_api/index.adoc#rest-api-index[REST API].
++
+----
+https://api.<cluster-id>.openshift.com
+----
+
+* Registry URL: The {product-title} URL for the private
+xref:../architecture/infrastructure_components/image_registry.adoc#architecture-infrastructure-components-image-registry[image
+registry]. In addition to containing all images used by {product-title}, `docker
+pull` and `docker push` can be used directly on the registry.
++
+----
+https://registry.<cluster-id>.openshift.com
+----
+
+* Metrics API URL: The {product-title} URL for the
+link:https://docs.openshift.com/container-platform/3.9/install_config/cluster_metrics.html#cluster-metrics-accessing-hawkular-metrics-directly[Hawkular
+Metrics API].
++
+----
+https://metrics.<cluster-id>.openshift.com
+----
+
+* Logging URL: The {product-title} URL for the aggregate logging Kibana interface.
++
+----
+https://logs.<cluster-id>.openshift.com
+----
+
+* If a callback URL is necessary:
++
+----
+https://api.<cluster-id>.openshift.com/oauth2callback/<IdP name>
+----
+
+[[access-your-cluster-dedicated-portal]]
+== Access your {product-title} portal
+
+Additional information about your cluster, including usage,
+subscription information, and past and upcoming maintenance
+is available at link:https://dedicated.openshift.com[].
+
+include::getting_started/topics/configuring_AWS_VPC.adoc[]
+
+[[access-your-cluster-status-updates]]
+== Receive status updates
+
+Access the status portal at link:https://status-dedicated.openshift.com[]. You
+can also subscribe to notifications via email, SMS, or RSS by changing your
+preferences in the status portal.
+
+[[access-your-cluster-support]]
+== Request support
+
+If you have questions about your environment or need to open a support ticket,
+you can open or view a support case in the
+link:https://access.redhat.com/support/cases/#/case/list[Red Hat Customer
+Portal].
+
+[[access-your-cluster-next-steps]]
+== Next steps
+
+You can download the {product-title} command line tools from your cluster’s
+xref:../architecture/infrastructure_components/web_console.adoc#web-console-cli-downloads[web
+console]. For help getting started with command line tools, see the
+xref:../cli_reference/get_started_cli.adoc#cli-reference-get-started-cli[Get
+Started with the CLI guide]. You can also visit the
+xref:../developers_console.adoc#getting-started-developers-console[Getting
+Started Guide for developers].
```

Comment:
The link for `Getting Started Guide for developers` can not be reached. Is the doc not ready?

Reply:
Hmm, maybe it needs to be `xref:developers_console.adoc`? Since they're both under the `getting_started` section?

Reply:
@wgordon17 @yasun1 I updated the path. PTAL.